context/app/static/js/components/detailPage/ProcessedData/HelperPanel/HelperPanel.tsx

```diff
@@ -110,7 +111,9 @@ function HelperPanelActions() {
           </SecondaryBackgroundTooltip>
         }
         datasetDetails={{ hubmap_id, uuid, status }}
+        dialogType="ADD_DATASETS_FROM_HELPER_PANEL"
       />
+      <AddDatasetsFromDetailDialog uuid={uuid} dialogType="ADD_DATASETS_FROM_HELPER_PANEL" />
```

Comment:
Can we just move this into `ProcessedDataWorkspaceMenu`?

Reply:
Yes!